jhotdraw-core/src/main/java/org/jhotdraw/draw/DefaultDrawing.java

```diff
@@ -131,11 +131,11 @@ public Figure findFigureExcept(Point2D.Double p, Figure ignore) {
   }
 
   @Override
-  public Figure findFigureBehind(Point2D.Double p, Figure figure) {
+  public Figure findFigureBehind(Point2D.Double p, Figure figure, double scaleDenominator) {
     boolean isBehind = false;
     for (Figure f : getFiguresFrontToBack()) {
```

Comment:
The "for" loop in the method "findFigureBehind(Point2D.Double p, Figure figure, double scaleDenominator)" is considerably similar to the first "for" loop of the method "findFigureBehind(Point2D.Double p, Object obj)". This is a code duplication case that I would suggest to address by extracting the common functionality into a separate method and reuse it for better maintainability.